dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/cache/DefaultCacheProvider.java

```diff
@@ -50,6 +50,40 @@
 public class DefaultCacheProvider implements CacheProvider
 {
 
+    /**
+     * Enum is used to make sure we do not use same region twice. Each method
+     * should have its own constant.
+     */
+    @SuppressWarnings( "squid:S115" ) // allow non enum-ish names
+    private enum Region
+    {
+        analyticsResponse,
```

Comment:
These should ideally have been capitalized as they are enum members. But I assume it was done to maintain the implementation and return the same string when the toString() is invoked? We could still change it though, only for redis based instances, the cache survives between restarts/redeployments etc. But we will eventually have to clear redis store as well on migrations.  Not really a major thing, so you and Lars can decide.

Reply:
Yes, the names are this way to keep the region keys as they were before.